tools/test-harnesses/src/utils/getQueries/index.ts

```diff
@@ -5,17 +5,25 @@ import { getDocument } from '../getDocument';
 
 const baseElement = getDocument();
 
-// Following [react-testing-libraby](https://github.com/testing-library/react-testing-library/blob/edb6344d578a8c224daf0cd6e2984f36cc6e8d86/src/pure.js#L195)
-const queriesForElements = getQueriesForElement(
-  baseElement.body,
-  // @ts-expect-error: `getQueriesForElement` is expecting all the queries outlined in this [file](https://github.com/testing-library/dom-testing-library/blob/main/types/get-queries-for-element.d.ts) but we're only passing in testid queries.
-  defaultQueries,
-);
-
-const {
-  getByTestId: getByLgId,
-  findByTestId: findByLgId,
-  queryByTestId: queryByLgId,
-} = queriesForElements;
-
-export { findByLgId, getByLgId, queryByLgId };
+let findByLgId;
+let getByLgId;
+let queryByLgId;
```

Comment:
Forking return/export values with a `let` declaration is a bit of a code-smell for me.

I would wrap this all in a function, something like this:
```typescript
function getBaseQueries() {
  const baseElement = getDocument();

  if (!baseElement) {
    return {
      findByLgId: undefined,
      getByLgId: undefined,
      queryByLgId: undefined,
    };
  }

  if (baseElement) {
    return getQueriesForElement(...);
  }
}

const queries = getBaseQueries();
export { queries };
``

Reply:
A benefit of this is that TS knows that if one query is undefined, they all are

Reply:
updated!